packages/core/admin/admin/src/content-manager/pages/ListView/components/TableRows/index.js

```diff
@@ -55,11 +55,7 @@ export const TableRows = ({
   const ReviewWorkflowsStage = useEnterprise(
     REVIEW_WORKFLOW_COLUMNS_CE,
     async () =>
-      (
-        await import(
-          '../../../../../../../ee/admin/content-manager/pages/ListView/ReviewWorkflowsColumn'
-        )
-      ).ReviewWorkflowsStageEE,
+      import('../../../../../../../ee/admin/content-manager/pages/ListView/ReviewWorkflowsColumn'),
```

Comment:
Is that not a named export? Can we make it one? :) Also: you should `await import`